subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -1695,28 +1695,49 @@ static int cmd_wifi_set_rts_threshold(const struct shell *sh, size_t argc, char
 
 	context.sh = sh;
 
-	if (strcmp(argv[1], "off") != 0) {
-		long rts_val = shell_strtol(argv[1], 10, &err);
+	if (argc > 2) {
+		PR_WARNING("Invalid number of arguments\n");
+		return -ENOEXEC;
+	}
 
-		if (err) {
-			shell_error(sh, "Unable to parse input (err %d)", err);
-			return err;
+	if (argc == 1) {
```

Comment:
may be add support for `show`?

Reply:
Ok will add like `wifi rts_threshold show`

Reply:
@krish2718 , as we are adding `show`. I think we can add `wifi rts_threshold set value/off` to setting value. Right?

Reply:
Either way works for me, we can leave it as `no_args == show` and `arg == value`.